src/Microsoft.Android.Templates/android-wear/.template.config/template.json

```diff
@@ -4,7 +4,7 @@
   "classifications": [ "Android", "Mobile" ],
   "identity": "Microsoft.Android.AndroidWearApp",
   "name": "Android Wear Application",
-  "description": "A project for creating a .NET Android Wear application",
+  "description": "A project for creating a .NET for Android Wear application",
```

Comment:
Does this feel weird? ".NET for Android Wear" sounds like a product name.

Reply:
I only "semantically read" the changes I made; I didn't *fully and deeply* read the changes. :-)

Yes, that feels weird.

Also, it's now ["Wear OS"](https://wearos.google.com).

We could instead say:

```
A project for creating Wear OS apps using .NET for Android.
```

which I think reads much better, but then makes me think: should this same phrasing apply to the `dotnet new android` template description?

```
A project for creating Android apps using .NET for Android.
```

which feels a tad duplicative?

Not sure how consistent vs. inconsistent we should be here.

Reply:
I think it would be better to have the one-off:
```
A project for creating a Wear OS application using .NET for Android.
```
And the other templates could stay the same.